test/E2ETest/testassets/projects/health-checks/tye-all.yaml

```diff
@@ -0,0 +1,36 @@
+# tye application configuration file
+# read all about it at https://github.com/dotnet/tye
+name: health-checks
+services:
+  - name: all
+    project: api/api.csproj
+    replicas: 3
+    bindings:
+      - port: 8004
+    liveness:
```

Comment:
These 2 probes are bigger than the entire yaml, that makes me 😢

Reply:
@davidfowl yeah... I thought it's best to go with something flexible like in k8s, but you can get a way with just providing a `path`, the rest have default values